client/package.json

```diff
@@ -1,11 +1,18 @@
 {
-  "name": "client",
+  "name": "@waterpark/client",
   "version": "0.1.0",
+  "description": "Web client for Loo Labs' centralized clubs and events platform",
+  "main": "index.js",
+  "repository": "https://github.com/loolabs/waterpark",
+  "author": "Loo Labs",
+  "license": "MIT",
   "private": true,
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start"
+    "lint": "exit 0",
```

Comment:
Why do we have no linting for `client`?

Reply:
it felt out of my jurisdiction to force prettier on the FE; that's why prettier is only installed in the BE

Reply:
I think adding prettier to the client in this PR is ok